packages/@jsii/kernel/lib/api.ts

```diff
@@ -220,44 +220,46 @@ export interface NamingRequest {
 }
 
 export interface NamingResponse {
-  readonly naming: { readonly [language: string]: { readonly [key: string]: any } | undefined };
+  readonly naming: {
+    readonly [language: string]: { readonly [key: string]: any } | undefined;
+  };
```

Comment:
I find it odd that it applies to `[lamguage: string]` but not to `[key: string]`... I feel like they should be treated the same.